docs/gettingstarted/mk8s/k8s-microk8s-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Clustering SNMP with microk8s
+
+## Master node
+
+1. Setup master node using this [guide](./k8s-microk8s.md). 
+
+2. Generate joining token for replicas:
+
+```bash
+microk8s add-node
+```
+
+3. After running `add-node` on stdout you will got a commamd (like `microk8s join <master_node>:25000/<token>`) that you need to remember.
+
+## Replicas nodes
+
+1. Installing microk8s on replica node: 
+
+```bash
+sudo snap install microk8s --classic --channel=1.30/stable
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+su - $USER
+```
+
+2. After run `add-node` on **master node** you will got on stdout command that you need to run on your **replica node**:
+
+```bash
+microk8s join <master_node>:25000/<token>
+```
+
+3. Check that replicas joined cluster:
+
+```bash
+microk8s kubectl get nodes
+```
+
+
+## Scaling SC4SNMP
+
+When SC4SNMP will be installed ([guide you can find here](../sc4snmp-installation.md)) you can start to scale SC4SNMP.
+
+1. Add `worker` section on `values.yaml`:
+
+```yaml
+worker:
+  poller:
+    replicaCount: 4
+  trap:
+    replicaCount: 4
+  sender:
+    replicaCount: 4
+```
+
+2. Add `traps` replica count in `values.yaml`:
+
+```yaml
+traps:
+  replicaCount: 4
+```
+
+3. Redeploy SC4SNMP:
+
+```bash
+microk8s helm3 upgrade --install snmp -f values.yaml splunk-connect-for-snmp/splunk-connect-for-snmp --namespace=sc4snmp --create-namespace
+```
+
+4. Check that SC4SNMP has been scaled:
+
+```bash
+microk8s kubectl get pods -o wide -n sc4snmp
+```
+
+5. If you need autoscaling then explore this [configuration](../../configuration/worker-configuration.md#worker-scaling)
```

Comment:
maybe we can move the autoscaling instruction here, and in the worker instruction refer to this part. I think that would be more intuitive to have about scaling in one place.